src/overlays/actors/ovl_En_Dragon/z_en_dragon.c

```diff
@@ -725,7 +725,7 @@ void EnDragon_Dead(EnDragon* this, PlayState* play) {
 
 void EnDragon_UpdateDamage(EnDragon* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
-    u32 sp30;
+    u32 playerImpactType;
```

Comment:
Can you use the enum type and still match?
```suggestion
    PlayerImpactType playerImpactType;
```

Reply:
You can yeah. It's weird because `play->actorCtx.playerImpact.type` is `u8` so you're casting back and forth between `PlayerImpactType` and `u8`, but it doesn't seem to give warnings? So I put it in, let me know how it looks

Reply:
Even if it is an `u8`, the fact that an `u32` is required for the function declaration makes me think they may have actually used the enum type

Reply:
There's no more `u32`, even the `u32*` pointer was switched to `PlayerImpactType*` without any matching issues

Reply:
I mean, the `u32*` we had before this was changed to `PlayerImpactType*`, since a `u32*` was required instead of a `u8*` makes me think an enum type is fine for this